app/poros/github_pull_requests.rb

```diff
@@ -0,0 +1,10 @@
+class GithubPullRequests
+
+  def self.merged_pulls
+    github_pull_requests = GithubService.closed_pulls
+    github_pull_requests.count do |pr|
```

Comment:
In reality, I'd probably throw out the variable here to streamline. Nit-picky and def a style question.